code/modules/mob/new_player/login.dm

```diff
@@ -29,4 +29,4 @@
 	new_player_panel()
 	client.playtitlemusic()
 	if(ticker.current_state < GAME_STATE_SETTING_UP)
-		src << "Please set up your character and select \"Ready\". The game will start in about [round(ticker.GetTimeLeft()/10)] seconds."
\ No newline at end of file
+		src << "Please set up your character and select \"Ready\". The game will start in about [round((ticker.GetTimeLeft() + 1)/10)] seconds."

```

Comment:
the real answer here is that round() is a floor() if you don't give it a second arg, so just make it a real round by doing round(value, 1)